### PR TITLE
DOC-2951 patch release info added 4.5.5 and 4.8.1

### PR DESCRIPTION
**Pull request by benpesterr3**

### CODE REVIEW

File: content/en/docs/corda-enterprise/4.8/release-notes-enterprise.md [hunk 1]
```diff
@@ -15,6 +15,31 @@ weight: 1
 
 # Corda Enterprise release notes
 
+## Corda Enterprise 4.8.1
+
+Corda Enterprise 4.8.1 is a patch release of Corda Enterprise that fixes a security vulnerability in Corda Enterprise 4.5.3.
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.8/release-notes-enterprise.md:20]:
```suggestion
Corda Enterprise 4.8.1 is a patch release of Corda Enterprise that fixes a security vulnerability in Corda Enterprise 4.8.
```

File: content/en/docs/corda-enterprise/4.8/release-notes-enterprise.md [hunk 2]
```diff
@@ -15,6 +15,31 @@ weight: 1
...
+As a node operator, you should upgrade to the [latest released version of Corda](https://docs.corda.net/docs/corda-enterprise/index.html) if the fixed issues listed below are relevant to your work.
+
+### Fixed issues
+
+In this patch release:
+
+* A fix has been introduced to reduce memory consumption during batched transaction resolution of large backchains.
+* Support has been introduced for RedHat Enterprise Linux 8.x in Corda 4.8.1.
+* Performance verification of the HA Notary with CockroachDB 20.2.8 has been carried out - performance is not worse in comparison to the HA Notary configured on CockroachDB 20.1.6.
+* Support for Oracle 19C database has been validated.
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.8/release-notes-enterprise.md:35]: Can you pls move the Oracle item to the top of the list - this is a top priority thing for one of our big customers so I'd have it on top.

File: content/en/docs/corda-enterprise/4.8/release-notes-enterprise.md [hunk 3]
```diff
@@ -15,6 +15,31 @@ weight: 1
...
+* Support for PostgreSQL 13.3 for the Node in 4.8.1.
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.8/release-notes-enterprise.md:36]:
```suggestion
* Support for PostgreSQL 13.3 for node databases has been added.
```

File: content/en/docs/corda-enterprise/4.8/release-notes-enterprise.md [hunk 4]
```diff
@@ -15,6 +15,31 @@ weight: 1
...
+* Hibernate ORM has been updated to version to 5.4.32 to remove a security concern.
+* The Node console configuration has been updated. Configuration is now set in `node.management.plugin.middleware`, no longer `node.admin.middleware`.
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.8/release-notes-enterprise.md:38]:
```suggestion
* The Node management console configuration has been updated. Configuration is now set in `node.management.plugin.middleware`, no longer `node.admin.middleware`.
```

Comment (ivanterziev-r3) [reply content/en/docs/corda-enterprise/4.8/release-notes-enterprise.md:38]: Perhaps add a link to the node management console page too.

File: content/en/docs/corda-enterprise/4.5/release-notes-enterprise.md [hunk 5]
```diff
@@ -16,6 +16,23 @@ weight: 1
...
+
+As a developer, you should upgrade to the [latest released version of Corda](https://docs.corda.net/docs/corda-enterprise/index.html) as soon as possible. Check the latest Corda Enterprise release notes and upgrade guide [here](https://docs.corda.net/docs/corda-enterprise/release-notes-enterprise.html).
+
+As a node operator, you should upgrade to the [latest released version of Corda](https://docs.corda.net/docs/corda-enterprise/index.html) if the fixed issues listed below are relevant to your work.
+
+### Fixed issues
+
+In this patch release:
+
+* Corda dependency vulnerability CVE-2020-28052 was fixed.
```
Comment (ivanterziev-r3) [inline content/en/docs/corda-enterprise/4.5/release-notes-enterprise.md:33]: was or has been (for consistency)

**Review by ivanterziev-r3 — CHANGES_REQUESTED** [review]
Looks good, added a few minor comments/suggestions.

File: content/en/docs/corda-enterprise/4.5/release-notes-enterprise.md (hunk 5 above)
Comment (benpesterr3) [inline content/en/docs/corda-enterprise/4.5/release-notes-enterprise.md:33]:
```suggestion
* Corda dependency vulnerability CVE-2020-28052 has been fixed.
```

File: content/en/docs/corda-enterprise/4.8/release-notes-enterprise.md (hunk 2 above)
Comment (benpesterr3) [inline content/en/docs/corda-enterprise/4.8/release-notes-enterprise.md:35]:
```suggestion
* Support for Oracle 19C database has been validated.
* A fix has been introduced to reduce memory consumption during batched transaction resolution of large backchains.
* Support has been introduced for RedHat Enterprise Linux 8.x in Corda 4.8.1.
* Performance verification of the HA Notary with CockroachDB 20.2.8 has been carried out - performance is not worse in comparison to the HA Notary configured on CockroachDB 20.1.6.
```

File: content/en/docs/corda-enterprise/4.8/release-notes-enterprise.md [hunk 6]
```diff
@@ -15,6 +15,31 @@ weight: 1
...
+
+In this patch release:
+
+* Support for Oracle 19C database has been validated.
+* A fix has been introduced to reduce memory consumption during batched transaction resolution of large backchains.
+* Support has been introduced for RedHat Enterprise Linux 8.x in Corda 4.8.1.
+* Performance verification of the HA Notary with CockroachDB 20.2.8 has been carried out - performance is not worse in comparison to the HA Notary configured on CockroachDB 20.1.6.
+* Support for PostgreSQL 13.3 for node databases has been added.
+* Hibernate ORM has been updated to version to 5.4.32 to remove a security concern.
+* The Node management console configuration has been updated. Configuration is now set in `node.management.plugin.middleware`, no longer `node.admin.middleware`.
```
Comment (erinking-r3) [inline content/en/docs/corda-enterprise/4.8/release-notes-enterprise.md:38]:
```suggestion
* The [Node management console](https://docs.corda.net/docs/corda-enterprise/4.8/node/management-console.html#node-management-console) configuration has been updated. Configuration is now set in `node.management.plugin.middleware`, no longer `node.admin.middleware`.
```

Comment (erinking-r3) [reply content/en/docs/corda-enterprise/4.8/release-notes-enterprise.md:38]: This is also true for the flow management console

```suggestion
* The Flow management console configuration has been updated. Configuration is now set in `flow.management.plugin.middleware`, no longer `flow.admin.middleware`.
```

Comment (erinking-r3) [reply content/en/docs/corda-enterprise/4.8/release-notes-enterprise.md:38]: Please also add 

* The [Flow management console](https://docs.corda.net/docs/corda-enterprise/4.8/node/node-flow-management-console.html#flow-management-console) configuration has been updated. Configuration is now set in `flow.management.plugin.middleware`, no longer `flow.admin.middleware`.

File: content/en/docs/corda-enterprise/4.8/release-notes-enterprise.md [hunk 7]
```diff
@@ -15,6 +15,32 @@ weight: 1
...
+In this patch release:
+
+* Support for Oracle 19C database has been validated.
+* A fix has been introduced to reduce memory consumption during batched transaction resolution of large backchains.
+* Support has been introduced for RedHat Enterprise Linux 8.x in Corda 4.8.1.
+* Performance verification of the HA Notary with CockroachDB 20.2.8 has been carried out - performance is not worse in comparison to the HA Notary configured on CockroachDB 20.1.6.
+* Support for PostgreSQL 13.3 for node databases has been added.
+* Hibernate ORM has been updated to version to 5.4.32 to remove a security concern.
+* The [Node management console](https://docs.corda.net/docs/corda-enterprise/4.8/node/management-console.html#node-management-console) configuration has been updated. Configuration is now set in `node.management.plugin.middleware`, no longer `node.admin.middleware`.
+The [Flow management console](https://docs.corda.net/docs/corda-enterprise/4.8/node/node-flow-management-console.html#flow-management-console) configuration has been updated. Configuration is now set in flow.management.plugin.middleware, no longer flow.admin.middleware.
```
Comment (benpesterr3) [inline content/en/docs/corda-enterprise/4.8/release-notes-enterprise.md:39]:
```suggestion
The [Flow management console](https://docs.corda.net/docs/corda-enterprise/4.8/node/node-flow-management-console.html#flow-management-console) configuration has been updated. Configuration is now set in `flow.management.plugin.middleware`, no longer `flow.admin.middleware`.
```

File: content/en/docs/corda-enterprise/4.8/release-notes-enterprise.md [hunk 8]
```diff
@@ -15,6 +15,32 @@ weight: 1
...
+
+* Support for Oracle 19C database has been validated.
+* A fix has been introduced to reduce memory consumption during batched transaction resolution of large backchains.
+* Support has been introduced for RedHat Enterprise Linux 8.x in Corda 4.8.1.
+* Performance verification of the HA Notary with CockroachDB 20.2.8 has been carried out - performance is not worse in comparison to the HA Notary configured on CockroachDB 20.1.6.
+* Support for PostgreSQL 13.3 for node databases has been added.
+* Hibernate ORM has been updated to version to 5.4.32 to remove a security concern.
+* The [Node management console](https://docs.corda.net/docs/corda-enterprise/4.8/node/management-console.html#node-management-console) configuration has been updated. Configuration is now set in `node.management.plugin.middleware`, no longer `node.admin.middleware`.
+The [Flow management console](https://docs.corda.net/docs/corda-enterprise/4.8/node/node-flow-management-console.html#flow-management-console) configuration has been updated. Configuration is now set in `flow.management.plugin.middleware`, no longer `flow.admin.middleware`.
+.
```
Comment (benpesterr3) [inline content/en/docs/corda-enterprise/4.8/release-notes-enterprise.md:40]:
```suggestion

```

**Review by ivanterziev-r3 — APPROVED** [review]
LGTM

**Pull request merged**